blas/unit_test/Test_Blas2_gemv.hpp

```diff
@@ -96,8 +103,11 @@ void impl_test_gemv(const char* mode, int M, int N) {
   Kokkos::deep_copy(h_y, y);
   int numErrors = 0;
   for (int i = 0; i < ldy; i++) {
-    if (KAT_Y::abs(expected(i) - h_y(i)) > KAT_Y::abs(eps * expected(i)))
+    if (KAT_Y::abs(expected(i) - h_y(i)) > tol) {
       numErrors++;
+      std::cout << "expected(i)=" << expected(i) << ", h_y(i)=" << h_y(i)
```

Comment:
Consider appending 'ERROR: ' and printing to stderr, or, better yet, using a `kk_error` print utility formatted in a standardized way similar to: https://github.com/kokkos/kokkos-kernels/pull/1661/files#diff-7e3381de82f3f74d36caf35bb386cb950fde00a4a980d84c944d1940372b6633R73.

Reply:
This suggestion can wait. I filed https://github.com/kokkos/kokkos-kernels/issues/1705.